engine/phone/scripts/brent.asm

```diff
@@ -11,11 +11,11 @@ BrentPhoneCalleeScript:
 	iftrue BrentMondayMorning
 
 .NotMonday:
-	farsjump UnknownScript_0xa0998
+	farsjump BrentHangupScript
```

Comment:
HangUp, not Hangup, for all of these `*HangupScript`s

Reply:
`Hangup` seems to be more common than `HangUp`, and I like it better. The only `HangUp` is `MomPhoneHangUpScript`, so I'll just change that one, if it's fine with you.

Reply:
It's just not grammatical. You hang up the phone; you have a hangup or hang-up about calling people on the phone at all. :P

Reply:
Gotcha.